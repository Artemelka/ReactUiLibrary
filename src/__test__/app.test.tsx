import React from 'react';

describe('Test App component', () => {
    test('Rendering snapshot', () => {
        expect(true).toBe(true);
    });
});
