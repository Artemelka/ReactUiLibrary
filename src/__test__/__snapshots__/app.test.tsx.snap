// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App component Rendering snapshot 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Connect(Provider)
    fetchDictionary={[Function]}
  >
    <Connect(ConnectedRouterWithContext)
      history={
        Object {
          "action": "POP",
          "block": [Function],
          "createHref": [Function],
          "go": [Function],
          "goBack": [Function],
          "goForward": [Function],
          "length": 1,
          "listen": [Function],
          "location": Object {
            "hash": "",
            "pathname": "/",
            "search": "",
            "state": undefined,
          },
          "push": [Function],
          "replace": [Function],
        }
      }
    >
      <div
        className="App"
      >
        <Route
          component={[Function]}
          exact={true}
          key="/"
          path="/"
        />
        <Route
          component={[Function]}
          key="/library"
          path="/library"
        />
        <Route
          component={[Function]}
          key="/layout"
          path="/layout"
        />
        <Route
          component={[Function]}
          key="/translate-editor"
          path="/translate-editor"
        />
      </div>
    </Connect(ConnectedRouterWithContext)>
  </Connect(Provider)>
</Provider>
`;
